es6-lib/services/summary.js

```diff
@@ -25,8 +25,24 @@ class SummaryService {
     //;_; see the note in service/spatial for why
     //this exists
     var onErr = _.once((err) => {
-      req.log.error(err.toJSON(), "Failed to generate summary");
-      return res.status(err.status()).send(err.toJSON());
+      var msg;
```

Comment:
eh don't really need to change this since it's just matching the code that is already there, but we really shouldn't be using `var` in es6 stuff.